lib/sw-precache.js

```diff
@@ -183,7 +183,7 @@ function generate(params, callback) {
     if (params.runtimeCaching) {
       var pathToSWToolbox = require.resolve('sw-toolbox/sw-toolbox.js');
       swToolboxCode = fs.readFileSync(pathToSWToolbox, 'utf8')
-        .replace('//# sourceMappingURL=sw-toolbox.map.json', '');
+        .replace('//# sourceMappingURL=./build/sw-toolbox.map.json', '');
```

Comment:
The version of `sw-toolbox.js` that gets installed via `npm` currently has the line `//# sourceMappingURL=./build/sw-toolbox.map.json` in it.

If that's not intentional then it may need to be updated in a `sw-toolbox` 3.2.2.